dynamic-config.md

```diff
@@ -0,0 +1,206 @@
+| schedule.hot-region-schedule-limit | 可以控制同时进行的热点调度的任务个数 |
+| schedule.hot-region-cache-hits-threshold | 用于设置 Region 被视为热点的阈值 |
+| schedule.high-space-ratio | 用于设置 store 空间充裕的阈值 |
+| schedule.low-space-ratio | 用于设置 store 空间不足的阈值 |
+| schedule.tolerant-size-ratio | 控制 balance 缓冲区大小 |
+| schedule.enable-remove-down-replica | 用于开启自动删除 DownReplica 的特性 |
+| schedule.enable-replace-offline-replica | 用于开启迁移 OfflineReplica 的特性 |
+| schedule.enable-make-up-replica | 用于开启补充副本的特性 |
+| schedule.enable-remove-extra-replica | 用于开启删除多余副本的特性 |
+| schedule.enable-location-replacement | 用于开启隔离级别检查 |
+| schedule.enable-cross-table-merge | 用于开启跨表 Merge |
+| schedule.enable-one-way-merge | 用于开启单向 Merge（只允许和下一个相邻的 Region Merge） |
+| replication.max-replicas | 用于设置副本的数量 |
+| replication.location-labels | 用于设置 TiKV 集群的拓扑信息 |
+| replication.enable-placement-rules | 开启 placement rules |
+| replication.strictly-match-label | 开启 label 检查 |
+| pd-server.use-region-storage | 开启独立的 region 存储 |
+| pd-server.max-gap-reset-ts | 用于设置最大的重置 timestamp 的间隔（BR）|
+| pd-server.key-type| 用于设置集群 key 的类型 |
+| pd-server.metric-storage | 用于设置集群 metrics 的存储地址 |
+| pd-server.dashboard-address | 用于设置 dashboard 的地址 |
+| replication-mode.replication-mode | 备份的模式 |
+
+具体参数意义可参考 [PD 配置文件描述](https://pingcap.com/docs-cn/stable/pd-configuration-file/)
+
+### TiKV
+
+| 参数 | 简介 |
+| --- | --- |
+| refresh-config-interval | 尝试更新配置的时间间隔 |
+| raftstore.sync-log | 数据、log 落盘是否 sync |
+| raftstore.raft-entry-max-size | 单个日志最大大小 |
+| raftstore.raft-log-gc-tick-interval | 删除 Raft 日志的轮询任务调度间隔时间 |
+| raftstore.raft-log-gc-threshold | 允许残余的 Raft 日志个数，软限制 |
+| raftstore.raft-log-gc-count-limit | 允许残余的 Raft 日志个数，硬限制 |
+| raftstore.raft-log-gc-size-limit | 允许残余的 Raft 日志大小，硬限制 |
+| raftstore.raft-entry-cache-life-time | 内存中日志 cache 允许的最长残留时间 |
+| raftstore.raft-reject-transfer-leader-duration | 控制迁移 leader 到新加节点的最小时间 |
+| raftstore.split-region-check-tick-interval | 检查 Region 是否需要分裂的时间间隔 |
+| raftstore.region-split-check-diff | 允许 Region 数据超过指定大小的最大值 |
+| raftstore.region-compact-check-interval | 检查是否需要人工触发 RocksDB compaction 的时间间隔 |
+| raftstore.region-compact-check-step | 每轮校验人工 compaction 时，一次性检查的 region 个数 |
+| raftstore.region-compact-min-tombstones | 触发 RocksDB compaction 需要的 tombstone 个数 |
+| raftstore.region-compact-tombstones-percent | 触发 RocksDB compaction 需要的 tombstone 所占比例 |
+| raftstore.pd-heartbeat-tick-interval | 触发 Region 对 PD 心跳的时间间隔 |
+| raftstore.pd-store-heartbeat-tick-interval | 触发 store 对 PD 心跳的时间间隔 |
+| raftstore.snap-mgr-gc-tick-interval | 触发回收过期 snapshot 文件的时间间隔 |
+| raftstore.snap-gc-timeout | snapshot 文件的最长保存时间 |
+| raftstore.lock-cf-compact-interval | 触发对 lock CF compact 检查的时间间隔 |
+| raftstore.lock-cf-compact-bytes-threshold | 触发对 lock CF 进行 compact 的大小 |
+| raftstore.messages-per-tick | 每轮处理的消息最大个数 |
+| raftstore.max-peer-down-duration | 副本允许的最长未响应时间 |
+| raftstore.max-leader-missing-duration | 允许副本处于无主状态的最长时间 |
+| raftstore.abnormal-leader-missing-duration | 允许副本处于无主状态的时间 |
```

Comment:
The two parameters look similar. What's their difference?

Reply:
```suggestion
| raftstore.max-leader-missing-duration | 允许副本处于无主状态的最长时间，超过将会向 PD 校验自己是否已经被删除 |
| raftstore.abnormal-leader-missing-duration | 允许副本处于无主状态的时间，超过将视为异常，标记在 metrics 和日志中 |
```